src/doc/trpl/choosing-guarantees.md

```diff
@@ -0,0 +1,417 @@
+% Wrapper Types
+
+// Alternate title: "Choosing your Guarantees"
+
+A recurring theme in Rust programming is that Rust gives one control over the costs and guarantees
+of their program.
```

Comment:
> A recurring theme in Rust programming is that Rust gives one control over the costs and guarantees
> of their program.

Maybe instead: "One important feature of Rust as language ist that it lets one control the costs and guarantees
of their program."

(By the way, is the rest of the book written in the "you/your" or "one/their" style?)